rest/route/user_test.go

```diff
@@ -1011,3 +1011,89 @@ func TestOffboardUserHandlerAdminis(t *testing.T) {
 		assert.NotContains(t, projRef.Admins, offboardedUser)
 	}
 }
+
+func TestGetUserHandler(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert.NoError(t, db.ClearCollections(user.Collection))
+	usrToRetrieve := user.DBUser{
+		Id:           "beep.boop",
+		DispName:     "robots are good",
+		EmailAddress: "bots_r@us.com",
+		APIKey:       "secret_key",
+		PatchNumber:  12,
+		OnlyAPI:      true,
+		SystemRoles: []string{
+			"bot_access",
+		},
+	}
+	me := user.DBUser{
+		Id:           "me",
+		EmailAddress: "i@rock.com",
+		APIKey:       "my_key",
+	}
+	assert.NoError(t, usrToRetrieve.Insert())
+	assert.NoError(t, me.Insert())
+
+	for testName, testCase := range map[string]func(t *testing.T){
+		"UserNotFound": func(t *testing.T) {
+			req, err := http.NewRequest(http.MethodGet, "http://example.com/api/rest/v2/users/no_one", nil)
+			req = gimlet.SetURLVars(req, map[string]string{"user_id": "no_one"})
+
+			require.NoError(t, err)
+			handler := makeGetUserHandler()
+
+			assert.NoError(t, handler.Parse(ctx, req))
+			assert.Equal(t, handler.(*getUserHandler).userId, "no_one")
+
+			resp := handler.Run(gimlet.AttachUser(context.Background(), &me))
+			assert.Equal(t, resp.Status(), http.StatusNotFound)
+		}, "UserFound": func(t *testing.T) {
+			req, err := http.NewRequest(http.MethodGet, "http://example.com/api/rest/v2/users/beep.boop", nil)
+			req = gimlet.SetURLVars(req, map[string]string{"user_id": "beep.boop"})
+
+			require.NoError(t, err)
+			handler := makeGetUserHandler()
+
+			assert.NoError(t, handler.Parse(ctx, req))
+			assert.Equal(t, handler.(*getUserHandler).userId, "beep.boop")
```

Comment:
nit: if we're going to type assert `handler.(*getUserHandler)`, can we put it in a comma ok like on L1064 so it doesn't panic?